tests/Integration/Behaviour/Features/Context/Domain/Product/UpdateProductSuppliersFeatureContext.php

```diff
@@ -256,7 +239,7 @@ public function assertDefaultSupplier(string $productReference, TableNode $table
             unset($data['default supplier reference']);
         }
 
-        Assert::assertEmpty($data, sprintf('Some provided product supplier fields haven\'t been asserted: %s', var_export($data, true)));
+        Assert::assertEmpty($data, sprintf('Some provided product supplier fields haven\'t been asserted: %s', implode(',', $data)));
```

Comment:
The implode output doesn't allow to understand what is left since it doesn't display the array keys That's why I replaced it with a `var_export` in most Context classes, it helps debugging more efficiently